dev-python/click/click-7.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python package for creating beautiful command line interfaces"
+SRC_URI="https://github.com/pallets/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="http://click.pocoo.org/ https://pypi.org/project/click/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+REQUIRED_USE="doc? ( || ( $(python_gen_useflags 'python2*') ) )"
```

Comment:
Why oh why do we need it?

Reply:
Actually I don't understand why is it here :-/

Reply:
Probably because only py2 allowed to build docs some ages ago?

Reply:
https://github.com/gentoo/gentoo/commit/de00f8e224cded2e844f11892a9eaea4c8251d33